docs/dev/developer-guides/building-on-zksync/events.md

```diff
@@ -14,30 +14,50 @@ At zkSync, events behave the same way as in Ethereum.
 ## Events filtering
 
 Filtering is used to query indexed data and provide lower-cost data storage when the data is not required to be accessed on-chain.
+While filtering, we advice to load events by block ranges (0-1999, 2000-3999, ...) and index the result on your end, else you will get an error that says "block range should be less than or equal to 2000".
```

Comment:
When filtering, you should load events by block ranges (0-1999, 2000-3999, ...) and index the result on your end, or else you will get an error that says "block range should be less than or equal to 2000".